development/build/index.js

```diff
@@ -3,6 +3,10 @@
 //
 // run any task with "yarn build ${taskName}"
 //
+global.globalThis = global
```

Comment:
What is the purpose of this line? :thinking:

Reply:
SES otherwise throws `ReferenceError: globalThis is not defined`

Reply:
node v10 does not have `globalThis` which is required by SES
node v2 does have it